assets/blocks/course-completed-actions/edit.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * WordPress dependencies
+ */
+import { InnerBlocks } from '@wordpress/block-editor';
+import { applyFilters } from '@wordpress/hooks';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Filter the block template.
+ *
+ * @param {Array} template Block template.
+ */
+const TEMPLATE = [
+	[
+		'core/buttons',
+		[],
+		applyFilters( 'sensei-lms.Course.completedActions', [
+			[
+				'core/button',
+				{
+					text: __( 'Find More Courses', 'sensei-lms' ),
```

Comment:
This will make this location static here once saved, but that is probably fine? For dynamic URLs (i.e. View Certificate), we might be able to change in the render function?

Reply:
Nope, don't like it. Reworked in https://github.com/Automattic/sensei/pull/4266/commits/8515a5306bcd14e7e8e2fcfe46b1feab2fd32edc.